src/Misc/Part.cpp

```diff
@@ -165,8 +165,18 @@ static const Ports partPorts = {
 
             //d.broadcast("/damage", "s", part_loc);
         }},
-
-
+        {"savexml:", rProp(internal) rDoc("Save Part to the file it has been loaded from"), 0,
+        [](const char *, RtData &d)
+        {
+            Part *p = (Part*)d.obj;
+            if (p->loaded_file[0] == '\0') {  // if part was never loaded or saved
+                time_t rawtime;     // make a new name from date and time
+                time (&rawtime);
+                const struct tm* timeinfo = localtime (&rawtime);
+                strftime (p->loaded_file,23,"%F_%R.xiz",timeinfo); 
```

Comment:
Do you think we should print the filename to console, so the user knows how it has been saved? Or do you think it is obvious?

I'm OK with both.